Mathlib/Analysis/SpecialFunctions/Log/Deriv.lean

```diff
@@ -303,8 +303,6 @@ theorem hasSum_log_one_add_inv {a : ℝ} (h : 0 < a) :
     · linarith
     · exact div_pos one_pos (by linarith)
   convert hasSum_log_sub_log_of_abs_lt_one h₁ using 1
-  have h₂ : (2 : ℝ) * a + 1 ≠ 0 := by linarith
-  have h₃ := h.ne'
```

Comment:
This one will semantically conflict with the field_simp rewrite: let's wait for that PR to land first.